src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -36,6 +42,37 @@
      * София - 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        Hashtable<String, Integer> employees = new Hashtable<>();
+        for (String value : names) {
+            employees.put(value, 0);
+        }
+
+        try {
+            DateFormat formatOfDate = new SimpleDateFormat("dd.mm.yyyy");
+            Date beginsFrom = formatOfDate.parse(dateFrom);
+            Date endsOn = formatOfDate.parse(dateTo);
+            if (beginsFrom.after(endsOn)) {
+                return null;
+            }
+            for (String dateNameIntInt : data) {
```

Comment:
dateNameIntInt what does it mean?

Reply:
That mean that every String have a format like date_name_int_int and will be parsed and splited soon

Reply:
intInt isn't very good name
try to create another